dev-resources/keycloak_demo/test-realm.json

```diff
@@ -1372,7 +1381,7 @@
       "subType" : "anonymous",
       "subComponents" : { },
       "config" : {
-        "allowed-protocol-mapper-types" : [ "saml-user-property-mapper", "oidc-address-mapper", "saml-role-list-mapper", "oidc-usermodel-attribute-mapper", "oidc-sha256-pairwise-sub-mapper", "oidc-full-name-mapper", "oidc-usermodel-property-mapper", "saml-user-attribute-mapper" ]
+        "allowed-protocol-mapper-types" : [ "saml-user-attribute-mapper", "oidc-full-name-mapper", "oidc-usermodel-attribute-mapper", "oidc-usermodel-property-mapper", "oidc-address-mapper", "oidc-sha256-pairwise-sub-mapper", "saml-user-property-mapper", "saml-role-list-mapper" ]
```

Comment:
What's going on here? It seems that all you did here was reorder the list entries.

Reply:
that probably happened on the re-export from keycloak

Reply:
Yeah that would make sense if the new realm file was auto-genned (imagine if you had to manually edit all the components!)

Reply:
Actually for the 0.0.0.0 -> localhost I did just that, but then there were more significant changes so I just used the UI